great_expectations/expectations/core/expect_column_to_exist.py

```diff
@@ -134,7 +135,7 @@ class ExpectColumnToExist(BatchExpectation):
             }}
     """  # noqa: E501
 
-    column: str
+    column: StrictStr = Field(min_length=1, description=COLUMN_DESCRIPTION)
     column_index: Union[int, SuiteParameterDict, None] = None
```

Comment:
We also show this field in the forms. Can you add a description to this one as well?